rescheduler/Makefile

```diff
@@ -1,7 +1,7 @@
 
 FLAGS =
 ENVVAR = GOOS=linux GOARCH=amd64 CGO_ENABLED=0
-REGISTRY = gcr.io/google-containers
+REGISTRY ?= gcr.io/google-containers
```

Comment:
Could you change the default value to gcr.io/k8s-image-staging?

Reply:
Done.